frontend/packages/dev-console/src/components/import/image-search/ImageStreamTagDropdown.tsx

```diff
@@ -118,6 +119,11 @@ const ImageStreamTagDropdown: React.FC<{ disabled?: boolean; formContextField?:
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [imageStream.tag]);
 
+  React.useEffect(() => {
+    reload && imageStream.tag && searchImageTag(imageStream.tag);
```

Comment:
Your reload value should not be a boolean. The hook won't run the next time user clicks on reload. I think it's better to maintain a `reloadCount` which gets incremented every time user clicks reload. We did something similar for `EditBuildConfigForm`.